server/src/main/java/io/druid/segment/loading/OmniSegmentLoader.java

```diff
@@ -52,12 +54,14 @@
   public OmniSegmentLoader(
```

Comment:
Having it become the `SegmentLocalCacheManager` sounds reasonable to me.  It definitely clarifies what it is actually doing.